.github/CHANGELOG.md

```diff
@@ -48,6 +48,10 @@
 * Changed the name of `lightning.tensor` to `default.tensor` with the `quimb` backend.
   [(#719)](https://github.com/PennyLaneAI/pennylane-lightning/pull/719)
 
+* Dispatch the C++ `Measurements.probs(wires)` method in Lightning-Qubit and Lighnting-Kokkos to `Measurements.probs()` when requesting the probabilities for all wires.
+  This will trigger a more optimized implementation for calculating the probabilities of the whole system.
+  [(#)]()
```

Comment:
```suggestion
* Patch the C++ `Measurements.probs(wires)` method in Lightning-Qubit and Lighnting-Kokkos to `Measurements.probs()` when called with all wires.
  This will trigger a more optimized implementation for calculating the probabilities of the entire system.
  [(#744)](https://github.com/PennyLaneAI/pennylane-lightning/pull/744)
```